### PR TITLE
feat: sandbox support

### PR DESCRIPTION
**Pull request by florian-hoenicke**

### CODE REVIEW

File: now/dialog.py [hunk 1]
```diff
@@ -147,7 +149,7 @@ def ask_data(user_input: UserInput, **kwargs):
 
 
 def prompt_plus(questions, attribute, **kwargs):
-    if kwargs and kwargs.get(attribute):
+    if kwargs and attribute in kwargs:
```
Comment (joschkabraun) [inline now/dialog.py:152]: This leads to None error. Using this resolves the issue:
if kwargs and kwargs.get(attribute):

**Review by joschkabraun — CHANGES_REQUESTED** [review]
After fixing None error, PR worked smothly for me. So can merge after that.

File: now/dialog.py (hunk 1 above)
Comment (joschkabraun) [reply now/dialog.py]: Or the actual solution as you mentioned in the thread.

Comment (florian-hoenicke) [reply now/dialog.py]: Thanks for finding this out. Done

File: tests/integration/test_end_to_end.py [hunk 2]
```diff
@@ -19,13 +20,17 @@ def test_backend(
     output_modality: str,
     dataset: str,
     quality: str,
+    sandbox: bool,
     cluster: str,
     cluster_new: str,
 ):
+    if sandbox and dataset != 'best-artworks':
+        pytest.skip("doesn't need to be tested")
```
Comment (makram93) [inline tests/integration/test_end_to_end.py:28]:
```suggestion
sandbox= False
if datset == deepfashion:
     sandbox=True

```

File: tests/integration/test_end_to_end.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 from now.dialog import NEW_CLUSTER
 
 
+@pytest.mark.parametrize('sandbox', [False, True])
```
Comment (makram93) [inline tests/integration/test_end_to_end.py:11]:
```suggestion
```

File: tests/integration/test_end_to_end.py [hunk 4]
```diff
@@ -19,13 +20,17 @@ def test_backend(
     output_modality: str,
     dataset: str,
     quality: str,
+    sandbox: bool,
```
[the remark this answers is not in the document]
Comment (florian-hoenicke) [reply tests/integration/test_end_to_end.py]: done, thanks

File: tests/integration/test_end_to_end.py (hunk 3 above)
Comment (florian-hoenicke) [reply tests/integration/test_end_to_end.py]: done, thanks

File: tests/integration/test_end_to_end.py (hunk 2 above)
Comment (florian-hoenicke) [reply tests/integration/test_end_to_end.py]: changed

**Pull request merged**